Dockerfile

```diff
@@ -39,6 +27,34 @@ COPY --chown=node ./ ./
 RUN sed -i "s|<<VERSION_REPLACED_WHEN_BUILT>>|${WORKBENCH_CLIENT_VERSION}|" ./src/environments/environment.prod.ts
 
 RUN npm run build:ssr
+
+
+
+FROM node:current-alpine
+
+WORKDIR /home/node/workbench-client
+
+LABEL maintainer="Charles Alleman <alleman@qut.edu.au>" \
+  description="Production environment for workbench client server" \
+  version=${WORKBENCH_CLIENT_VERSION} \
+  name="Workbench Client" \
+  vendor="QUT Ecoacoustics" \
+  url="https://github.com/QutEcoacoustics/workbench-client" \
+  vsc-url="https://github.com/QutEcoacoustics/workbench-client" \
+  ref=${GIT_COMMIT} \
+  schema-version="1.0"
+
+# drop privileges
+USER node
+
+RUN mkdir -p  /home/node/workbench-client
+WORKDIR /home/node/workbench-client
+
+COPY --from=BUILD_IMAGE /home/node/workbench-client/dist ./dist
+COPY --from=BUILD_IMAGE /home/node/workbench-client/package.json ./package.json
```

Comment:
we had a default environment file in the container i think? Is that in dist?

Reply:
Normally whenever i run the docker container, i provide the environment file. That said, you can find the environment file in the dist folder still, but its not guaranteed to work because its a dev file.